docs/nodes/consensus-node.mdx

```diff
@@ -143,53 +209,78 @@ cd $HOME<br/>
 rm -rf ~/.celestia-app/data<br/>
 mkdir -p ~/.celestia-app/data<br/>
 SNAP_NAME=$(curl -s https://snaps.qubelabs.io/celestia/ | \<br/>
-&nbsp; egrep -o ">{constants.mochaChainId}.*tar" | tr -d ">")<br/>
+    egrep -o ">{constants.mochaChainId}.*tar" | tr -d ">")<br/>
 wget -O - https://snaps.qubelabs.io/celestia/$&#123;SNAP_NAME&#125; | tar xf - \<br/>
-&nbsp; -C ~/.celestia-app/data/<br/>
+    -C ~/.celestia-app/data/<br/>
 </code></pre>
 
 </TabItem>
-<TabItem value="blockspacerace" label="Blockspace Race">
+<TabItem value="arabica" label="Arabica">
 
-Run the following command to quick-sync from a snapshot for `blockspacerace`:
+Run the following command to quick-sync from a snapshot for `arabica-10`:
 
-```bash
-cd $HOME
-rm -rf ~/.celestia-app/data
-mkdir -p ~/.celestia-app/data
-SNAP_NAME=$(curl -s https://snaps.qubelabs.io/celestia/ | \
-    egrep -o ">blockspacerace.*tar" | tr -d ">")
-wget -O - https://snaps.qubelabs.io/celestia/${SNAP_NAME} | tar xf - \
-    -C ~/.celestia-app/data/
-```
+<pre><code>
+cd $HOME<br/>
+rm -rf ~/.celestia-app/data<br/>
+mkdir -p ~/.celestia-app/data<br/>
+SNAP_NAME=$(curl -s https://snaps.qubelabs.io/celestia/ | \<br/>
+    egrep -o ">{constants.arabicaChainId}.*tar" | tr -d ">")<br/>
+wget -O - https://snaps.qubelabs.io/celestia/$&#123;SNAP_NAME&#125; | tar xf - \<br/>
+    -C ~/.celestia-app/data/<br/>
+</code></pre>
 
 </TabItem>
 </Tabs>
-````
+```
 
-### Start the celestia-app
+### Start the consensus node
 
-In order to start your validator node, run the following:
+In order to start your full consensus node, run the following:
+
+```sh
+celestia-appd start
+```
+
+Follow the tutorial on
+[setting up Celestia App as a background process with SystemD](./systemd.md).
```

Comment:
[optional] do we want to clarify that this is an optional step?

```suggestion
Optional: If you would like celestia-app to run as a background process, you can follow the [SystemD tutorial](./systemd.md).
```

Reply:
absolutely, thank you